src/h5web/visualizations/heatmap/AxisGrid.tsx

```diff
@@ -3,20 +3,16 @@ import { useThree, useFrame, Dom } from 'react-three-fiber';
 import { scaleLinear } from 'd3-scale';
 import IndexAxis from './IndexAxis';
 import styles from './HeatmapVis.module.css';
-import { Domain } from './store';
+import { useProps } from './hooks';
+import { Domain } from './models';
 
 interface AxisDomains {
   left: Domain;
   bottom: Domain;
 }
 
-interface Props {
-  dims: [number, number];
-  axisOffsets: [number, number];
-}
-
-function AxisGrid(props: Props): JSX.Element {
-  const { dims, axisOffsets } = props;
+function AxisGrid(): JSX.Element {
+  const { dims, axisOffsets } = useProps();
```

Comment:
`useProps` is a custom hook that allows access to the Heatmap's core props, `dims` and `data` (and `axisOffsets` for convenience). Internally it just returns the value of the heatmap's context.